db_books.rb

```diff
@@ -0,0 +1,25 @@
+require 'json'
+
+module BooksPersistence
+  def store_books(books)
+    data = books.map { |book| { title: book.title, author: book.author } }
+    File.write('./books.json', JSON.generate(data))
+  end
```

Comment:
- [ ] You guys are doing a good job so far, but I came across a bug when running your code as shown below It seems when storing a created book in the JSON file It also pushes other random book data with a null value and ends up misplacing the author and title of the book, so kindly make sure to fix this:
  ![Code_QClJU5NtIt](https://github.com/Rifat8080/school-library/assets/104968236/f5b737fa-a71c-4835-b72a-b63992e1ad15)